lib/gruff/base.rb

```diff
@@ -504,6 +504,14 @@ def hide_labels?
       @hide_line_markers
     end
 
+    def hide_left_label_area?
```

Comment:
Changes in base.rb are the same ones made in PR #450, which should make the PRs compatible for merging.